### PR TITLE
New package: rest-server-0.9.7

### PR DESCRIPTION
**Pull request by arete**
Signed-off-by: Gerardo Di Iorio <arete74@gmail.com>

### CODE REVIEW

File: srcpkgs/rest-server/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'rest-server'
+pkgname=rest-server
+version=0.9.7
+revision=1
+build_style=go
+go_import_path="github.com/restic/rest-server"
+go_package="${go_import_path}/cmd/rest-server"
+short_desc="An HTTP server that implements restic's REST backend API"
+maintainer="Gerardo Di Iorio <arete74@gmail.com>"
+license="BSD-2"
```
Comment (maxice8) [inline srcpkgs/rest-server/template]: BSD-2-Clause is the SPDX identifier.

File: srcpkgs/rest-server/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+checksum=922811cfc49879f7086fe559d25ec1bb4169b2c4a5df3559b499e6292f435661
+system_accounts="_restserver"
+_restserver_homedir="/var/lib/rest-server"
+_restserver_descr="Rest Server Daemon User"
+make_dirs="/var/lib/rest-server 0755 _restserver _restserver"
+conf_files="/etc/default/rest-server"
+
+
+do_check() {
+	:
```
Comment (maxice8) [inline srcpkgs/rest-server/template]: Please add a comment on why.

File: srcpkgs/rest-server/template [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+homepage="https://github.com/restic/rest-server"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=922811cfc49879f7086fe559d25ec1bb4169b2c4a5df3559b499e6292f435661
+system_accounts="_restserver"
+_restserver_homedir="/var/lib/rest-server"
+_restserver_descr="Rest Server Daemon User"
+make_dirs="/var/lib/rest-server 0755 _restserver _restserver"
+conf_files="/etc/default/rest-server"
+
+
```
Comment (maxice8) [inline srcpkgs/rest-server/template]: there are 2 spaces here.

File: srcpkgs/rest-server/template (hunk 1 above)
Comment (arete) [reply srcpkgs/rest-server/template]: fix, i have update the pull request

File: srcpkgs/rest-server/template (hunk 2 above)
Comment (arete) [reply srcpkgs/rest-server/template]: empty check, similar to other template.

File: srcpkgs/rest-server/template (hunk 3 above)
Comment (arete) [reply srcpkgs/rest-server/template]: fix, update pull request

**Pull request closed without merging**